tests/Controller/FeedControllerTest.php

```diff
@@ -38,18 +41,63 @@ public function testSubmitAndCreatePost(): void
         $this->assertTrue($newPost->getPostText() === 'testing post');
     }
 
-    public function testItCanEditAnExistingPost(): void
+    public function testSubmitAndCreateComment(): void
     {
         $testUser = $this->dogRepo->findOneByUsername('testUser');
         $this->client->loginUser($testUser);
-
-        // We will need to find a new form here to do the edit
-        // Posts exist for the testUser already and their content is set. Just need to update it
-        // Should assert that the db was updated
+        $testUserPost = $this->postRepo->findBy(['post_text' => 'Woof woofingtons'])[0];
+        $url = "/post/{$testUserPost->getId()}/add-comment";
+        $crawler = $this->client->request('GET', $url);
+        $form = $crawler->selectButton('Save')->form();
+        $form["comment_form[comment_text]"] = 'testing comment';
+        $crawler = $this->client->submit($form);
+        
+        $newComment = $this->commentRepo->findBy(['comment_text' => 'testing comment'])[0];
+        $this->assertTrue($newComment->getCommentText() === 'testing comment');
```

Comment:
Love this test. Good job. Maybe add some line breaks to make it more readable? Otherwise, looks rock solid.

Reply:
doneeeee.